scripts/pull-templates.py

```diff
@@ -14,7 +14,7 @@ def parse_options(args=None):
     parser = OptionParser(args)
     parser.add_option("-s", "--server", dest="server", default="http://openlibrary.org/", help="URL of the openlibrary website (default: %default)")
     parser.add_option("--template-root", dest="template_root", default="/upstream", help="Template root (default: %default)")
-    parser.add_option("--default-plugin", dest="default_plugin", default="upstream", help="Default plugin (default: %default)")
+    parser.add_option("--default-plugin", dest="default_plugin", default="/upstream", help="Default plugin(default: %default)")
```

Comment:
```suggestion
    parser.add_option(
        "--default-plugin",
        dest="default_plugin",
        default="/upstream",
        help="Default plugin(default: %default)",
    )
```
This is how the Python code formatter `psf/black` would reformat this line.